app/models/user.rb

```diff
@@ -1,6 +1,22 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  has_many :reviews
+  has_many :cakes # !
```

Comment:
I think dependent destroy should be here as the cakes we would like to be destroyed are those sold by the user